src/lib/walletLinking/fetchWalletDataFromGithub.ts

```diff
@@ -37,7 +37,9 @@ export async function batchFetchWalletDataFromGithub(
       const result = fileContents[i];
 
       if (result.content) {
-        const walletData = parseWalletLinkingDataFromReadme(result.content);
+        const walletData = await parseWalletLinkingDataFromReadme(
```

Comment:
similarly this doesn't need to be async, since this code only runs in pipelines and not on clientside and therefore doesn't benefit from dynamic loading